src/components/tab/tab.tsx

```diff
@@ -66,10 +66,10 @@ export class Tab {
 
     return (
       <Host
-        aria-expanded={toAriaBoolean(this.active)}
         aria-labelledby={this.labeledBy}
         id={id}
         role="tabpanel"
+        tabIndex={this.active ? 0 : -1}
       >
```

Comment:
Beautiful! ✨ 

JAWS had some trouble handling this event until it was placed into a div as a child of the Host. Would it be possible for us to add these attributes to a child `<div>`?

```ts
      <Host>
        <div
          aria-labelledby={this.labeledBy}
          id={id}
          role="tabpanel"
          tabIndex={this.active ? 0 : -1}
        >
          <section>
            <slot />
          </section>
        </div>
      </Host>
```

Reply:
Does JAWS not work with custom elements? I'm not sure if we could move that. I'd rather not if possible.

Reply:
I'm not sure what may be causing the issue, but don't think its the custom element since it seems we've got a few use cases in other components?

I put in the change before checking NVDA first, and it also affects NVDA. So between the two of them would account for the majority of screen reader users (over 80% of sr users).

Is there another option we could consider to mitigate for JAWS and NVDA users?

Some background on the tests and the JAWS transcripts:

### Test experience:
![tab-test](https://user-images.githubusercontent.com/5023024/180083297-02bb76ae-0928-49e0-91d7-dca8cd60e376.gif)


### JAWS transcript with attrs in the `Host`:
The content is not read back when focused, instead the title is (in this case 'Tab 1 Title' and its role 'Tab').
```
Tab 1 Title Tab selected 
1 of 3
Tab 1 Title 
Tab 
```


### And the `div`:
```
Tab 1 Title Tab selected 
1 of 3
Tab 1 Content
```

Reply:
I don't think we can move these:

```
aria-labelledby={this.labeledBy}
          id={id}
```

They won't be able to be found by the tab title element.

Reply:
Would this work?

```ts
 <Host aria-labelledby={this.labeledBy} id={id}>
    <div role="tabpanel" tabIndex={this.active ? 0 : -1}>
      <section>
        <slot />
      </section>
    </div>
  </Host>
```

This is the JAWS transcript with the above code (Also works with NVDA).
```
Tab 1 Title Tab selected 
1 of 3
Tab 1 Title 
Tab 1 Content
```

Reply:
According to W3c, those should all be on the same element. https://www.w3.org/WAI/ARIA/apg/example-index/tabs/tabs-manual.html

Reply:
This is one of those things where I start to wonder if we should follow w3c or the screen readers. I tend to defer to w3c so we are at least following the directions and ensuring the proper structure of code.

Reply:
Hear ya on that front. FWIW, the sample does have a note on it indicating it shouldn't be used in production environments due to support gaps with browsers and AT. Maybe this is one of those use cases?

Reply:
> This is one of those things where I start to wonder if we should follow w3c or the screen readers. I tend to defer to w3c so we are at least following the directions and ensuring the proper structure of code.

I think we'll have to rely on our best judgement for these cases since ARIA guidance doesn't seem to mention custom elements/shadow DOM (at least to my knowledge) and sometimes screen readers don't do the right thing in this context. I like the idea of being spec-compliant, but it's not much use if screen readers can't use it.

Reply:
I think my best judgement here would be to not move the `id` and `aria-labeledby` to an element within the shadowRoot since those IDs would not be able to reference the element outside of the shadowRoot.

```jsx
aria-labelledby={this.labeledBy}
          id={id}
```